VMBackup/main/guestsnapshotter.py

```diff
@@ -148,6 +150,8 @@ def snapshot_seq(self, sasuri, sasuri_index, meta_data):
                         key = meta['Key']
                         value = meta['Value']
                         headers["x-ms-meta-" + key] = value
+                if(CommonVariables.isSnapshotTTLEnabled in settings):
+                    headers["x-ms-snapshot-ttl-expiry-hours"] = settings[CommonVariables.isSnapshotTTLEnabled]
                 self.logger.log(str(headers))
```

Comment:
settings[CommonVariables.isSnapshotTTLEnabled] will return true/false. I believe the value to set for x-ms-snapshot-ttl-expiry-hours is 1.